pkgs/applications/misc/hfst-ospell/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, autoreconfHook
+, fetchFromGitHub
+, icu
+, libarchive
+, libxmlxx
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hfst-ospell";
+  version = "0.5.3";
+  buildInputs = [
```

Comment:
Nit: a bit unusual order, more common is `pname`, `version`, `src`, `nativeBuildInputs`, `buildInputs`, `meta`.